hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/pom.xml

```diff
@@ -154,28 +154,21 @@
         </configuration>
       </plugin>
       <plugin>
-        <groupId>org.apache.hadoop</groupId>
-        <artifactId>hadoop-maven-plugins</artifactId>
+        <groupId>org.xolstice.maven.plugins</groupId>
+        <artifactId>protobuf-maven-plugin</artifactId>
         <executions>
           <execution>
-            <id>compile-protoc</id>
-            <goals>
-              <goal>protoc</goal>
-            </goals>
+            <id>src-test-compile-protoc</id>
```

Comment:
Before this change, `YarnSecurityTestAMRMTokenProtos` was getting generated in "target/generated-sources" directory and has been treated as main src. But ideally it should be part of test only.
This should not be a problem I hope.